realm/src/main/java/io/realm/internal/Table.java

```diff
@@ -677,7 +677,8 @@ public long getPrimaryKey() {
             if (pkTable == null) return NO_PRIMARY_KEY; // Free table = No primary key
             long rowIndex = pkTable.findFirstString(PRIMARY_KEY_CLASS_COLUMN_INDEX, getName());
             if (rowIndex != NO_MATCH) {
-                cachedPrimaryKeyColumnIndex = pkTable.getRow(rowIndex).getLong(PRIMARY_KEY_FIELD_COLUMN_INDEX);
+                String pkColumnName = pkTable.getRow(rowIndex).getString(PRIMARY_KEY_FIELD_COLUMN_INDEX);
```

Comment:
What convention are you referring to? It looks like a normal Java variable name to me?